test/examples/SampleCrowdsale.test.js

```diff
@@ -111,4 +114,12 @@ contract('SampleCrowdsale', function ([owner, wallet, investor]) {
     const balanceAfterRefund = web3.eth.getBalance(investor);
     balanceBeforeInvestment.should.be.bignumber.equal(balanceAfterRefund);
   });
+
+  describe('when goal > cap', function () {
```

Comment:
nit: we can move the `HIGH_GOAL` variable to this scope

Reply:
Good point, done.